packages/babel-plugin-transform-spread/test/fixtures/spread/nested-array-with-hole/exec.js

```diff
@@ -0,0 +1,2 @@
+let arr = ['a', ...['b',,'c']];
```

Comment:
What happens if this isn't statically analyzable?

Reply:
It uses `toConsumableArray(spreadArgument)`, which does all the necessary checks to ensure that it's correct. `arrayWithoutHoles` is part of those checks.

Reply:
https://babeljs.io/repl#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&corejs=3.6&spec=false&loose=false&code_lz=DYUwLgBAhhC8EG0CMAaCA6TAPAukA&debug=false&forceAllTransforms=true&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Creact%2Cstage-0%2Cstage-2&prettier=false&targets=&version=7.14.4&externalPlugins=%40babel%2Fplugin-transform-typescript%407.14.4